components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementAdminService.java

```diff
@@ -183,8 +224,15 @@ public IdentityProvider getIdentityProvider(String federatedIdPName)
      */
     public RequestPathAuthenticatorConfig[] getAllRequestPathAuthenticators()
             throws IdentityApplicationManagementException {
-        applicationMgtService = ApplicationManagementService.getInstance();
-        return applicationMgtService.getAllRequestPathAuthenticators(getTenantDomain());
+
+        try {
+            applicationMgtService = ApplicationManagementService.getInstance();
+            return applicationMgtService.getAllRequestPathAuthenticators(getTenantDomain());
+        } catch (IdentityApplicationManagementException idpException) {
+            log.error("Error while retrieving all request path authenticators", idpException);
```

Comment:
since we know the tenantDomain using getTenantDomain() method. Shall we include that in the error log?